Session_3/module_3.py

```diff
@@ -1,19 +1,29 @@
 # import time
+import time
 from typing import List
 
 Matrix = List[List[int]]
 
 
 def task_1(exp: int):
-    pass
+    return lambda x: x ** exp
```

Comment:
As described in Task 1 requirements, you should create closure function, rather than implicit function with lambda